wordpress-mobile/WordPress-iOS
# [Jetpack Focus] Remove carousel & redesign landing screen

## Description
- Fixes #19226

**This PR does not:**
- Remove the background for the buttons at the bottom
- Animate bottom buttons
- Animate logos

| Light | Dark |
| - | - |
| ![WP Light](https://user-images.githubusercontent.com/2092798/188040770-2a65e825-c40f-4989-971c-43ccc99a25b0.png) | ![Simulator Screen Shot - iPhone 13 - 2022-09-02 at 15 55 09](https://user-images.githubusercontent.com/2092798/188227419-836c5a29-355d-44e2-b639-eef51a4e50ba.png) |

| iPad Light | iPad Dark |
| - | - |
| ![WP Light iPad](https://user-images.githubusercontent.com/2092798/188040835-573b24c9-510d-4393-b45a-7218258bd315.png) | ![Simulator Screen Shot - iPad (9th generation) - 2022-09-02 at 15 56 23](https://user-images.githubusercontent.com/2092798/188227447-2095853c-ed68-47db-9fb7-d987e6d26b41.png) |

## Testing

- The feature flag `newLandingScreen` must be set to `true`.
  - If using a debug build without access to code, you'll need to log into the app to set this feature flag before logging back out.
- The view should only appear in the WordPress app.

### To test:
1. Load the WordPress app after a clean install, or after logging out of an existing account
2. Observe the landing screen

### Items to test:
- The view does not appear in the Jetpack app
- The login / sign up + enter existing site address buttons work
- Light / dark modes
- Accessibility
- iPhone / iPad
- LTR / RTL
  - The brush stroke background graphic should change sides depending on language direction

## Regression Notes
1. Potential unintended areas of impact
    - Landing / login screen functionality

2. What I did to test those areas of impact (or what existing automated tests I relied on)
    - Manual tests

3. What automated tests I added (or what prevented me from doing so)
    - None as these are mostly visual changes

PR submission checklist:
